Flutter_TDD_Architecture_Course_notes.md

```diff
@@ -1 +1,278 @@
 
+## Notes for Flutter TDD Clean Architecture Course Videos
+
+In this course we create a Number Trivia App following test driven development (TDD). It shows us how we can use TDD in _any_ App we want to build.
+
+#### 1. Explanation & Project Structure
+> Video playlist: https://www.youtube.com/watch?v=KjE2IDphA_U&list=PLB6lc7nQ1n4iYGE_khpXRdJkJEp9WOech
+
+In the `first tutorial` an explanation is given of the importance of having all functions tested in our application.  
+It also explains the **clean architecture**.
+
+All the features of the application will be divided into three folders, **domain**, **data** and **presentation**.
+It is also explained how each of these folders works and what they will contain.
+
+<br />
+
+#### 2. Entities & Use Cases 
+> Video 2: https://www.youtube.com/watch?v=lPkWX8xFthE&list=PLB6lc7nQ1n4iYGE_khpXRdJkJEp9WOech&index=2
+
+In the `second tutorial` we start by adding the packages of the dependencies to the `pubspec.yaml` file, that will be needed to build the application.
```

Comment:
thanks for adding the name of the file where the dependencies need to be added. 👍